runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsConfig.java

```diff
@@ -358,4 +358,37 @@ default boolean collapseEntries()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "showProfitable",
+		name = "Show profitable high alch items",
+		description = "Configures whether or not to highlight profitable high alch items",
+		position = 28
+	)
+	default boolean showHighlightProfit()
```

Comment:
use what every other one does and have the value range also determine on/off with 0 being off and others being on